ubuntu_from_source/Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG UBUNTU_VERSION=20.04
+
+FROM ubuntu:$UBUNTU_VERSION
+
+LABEL maintainer.name="ROOT team"
+LABEL maintainer.email="root-dev@cern.ch"
+
+ENV LANG=C.UTF-8
+
+COPY packages packages
+
+RUN apt-get update -qq && \
+    ln -sf /usr/share/zoneinfo/UTC /etc/localtime && \
+    apt-get -y install $(cat packages) wget git && \
+    apt-get autoremove -y && \
+    apt-get clean -y && \
+    rm -rf /var/cache/apt/archives/* && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV ROOTSYS /opt/root
+ENV PATH $ROOTSYS/bin:$PATH
+ENV PYTHONPATH $ROOTSYS/lib:$PYTHONPATH
+ENV CLING_STANDARD_PCH none
+
+RUN echo $ROOTSYS/lib >> /etc/ld.so.conf && \
+    ldconfig

```

Comment:
it might be better to build with `-Dgnuinstall=ON` and avoid setting these environment variables (except for PYTHONPATH maybe, which is probably needed anyway). @Axel-Naumann @bellenot do you agree?

Reply:
Yes, if you are going to build from source, I highly recommend configuring with the options I suggested in https://github.com/root-project/root/commit/6f80a44fb42ef9e05feafe7f1232eae6fa182a26. That means sourcing of `thisroot.sh` is not needed, and no need to set `PYTHONPATH` either. ROOT will just work out of the box.

Reply:
I tried to apply this but got errors at the end of compilation such as:

```
/bin/sh: 1: cd: can't cd to /opt/root/build/lib/root
```

It probably is some silly mistake, but it takes ages to compile. I will take a look when I have more time, but feel free to make the change.

Reply:
`ROOTSYS` should not be set in the environment. I think the error with cd is because `ROOTSYS` is confusing ROOT. Same for `PATH` and `PYTHONPATH`. You don't need to set those in the environment for building ROOT and after install with the options I suggested they should not be necessary either.

Reply:
Changed can be found here: https://github.com/lobis/root-docker/pull/1 I think you should be able to edit this branch.

Reply:
> `ROOTSYS` should not be set in the environment. I think the error with cd is because `ROOTSYS` is confusing ROOT. Same for `PATH` and `PYTHONPATH`. You don't need to set those in the environment for building ROOT and after install with the options I suggested they should not be necessary either.

As you can see on https://github.com/lobis/root-docker/pull/1 I have removed those lines.

Reply:
You posted the link to the branch after my comment.

Reply:
> 

Yes sorry, what I meant is that this is not the case of the failure, as I had tested it earlier. I then created the branch for debugging.